frontend/src/components/cluster-list.tsx

```diff
@@ -135,15 +135,15 @@ class ClusterList extends Component<ClusterListProp, ClusterListState> {
 //   clustersListUpdateFunc: PropTypes.func
 // };
 
-const mapStateToProps = (state: RootState) => ({
+const  = (state: RootState) => ({
   globalServerSelected: state.servers.globalServerSelected,
   globalClustersList: state.clusters.globalClustersList,
   globalTornjakServerInfo: state.servers.globalTornjakServerInfo,
   globalErrorMessage: state.tornjak.globalErrorMessage,
 })
```

Comment:
I don't think this is valid TypeScript

Reply:
Yes, I took a look at the build logs, it mentions this spot exactly: https://github.com/spiffe/tornjak/actions/runs/14287040357/job/40519000631?pr=613#step:11:349